lib/api.js

```diff
@@ -480,19 +476,23 @@ function getTraceSettings (xtrace, options = {}) {
       source: 5,
       rate: 0,
       edge: false,
-      metadata: aob.Event.makeRandom(0)
+      metadata: aob.Event.makeRandom(0),
```

Comment:
It looks like `metadata` is still in here to serve a default further down, L484, but i don't see why it needs to be added to osettings.
My proposal would be to to remove metadata here and rewrite line 485:
`osettings.traceTaskId = aob.Event.makeRandom(0)`

Reply:
it needs to be added to osettings so that i don't have to change appoptics-bindings-node in lock step. it returns metadata (and not traceTaskId). the code you're flagging is only executed if there was an error getting settings from oboe. once that oboe has been released i can switch over. but in the meantime i want the agent to operate with or without a compatible version of bindings.

Reply:
ok